BlackNectar/BlackNectar/StoresTableViewController.swift

```diff
@@ -128,41 +132,66 @@ extension StoresTableViewController {
     
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         
-        return stores.count
+        return stores.notEmpty ? stores.count : 1
         
     }
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: "storeCell", for: indexPath) as? StoresTableViewCell else {
+        if stores.notEmpty {
```

Comment:
Now that there are multiple cells that are being created here, you should split these two paths into two separate functions.

`createNoResultsCell()`
`createStoreCell()`

That would greatly reduce the amount of code going on here.